### ISSUE TITLE
train/test infinite loss

### ISSUE THREAD

**Issue by zzzrpagliari** [quora/qmf#11]
It might be difficult to reproduce, but every now and then, both train and test loss are infinite from the beginning until the end of the process. 
Have you ever noticed that?

**Comment by albietz** [comment]
Hi, if this is BPR, did you try using a smaller learning rate? (`--init_learning_rate=<x>`)

**Comment by zzzrpagliari** [comment]
No, but I can try. Any chance a warning message can be raised under such circumstances?

**Comment by albietz** [comment]
Hi, I added an error message for such situations. Let us know if a smaller learning rate fixes your issues.

**Comment by zzzrpagliari** [comment]
Hi,
Yes that worked.


Thank you !
